setup-build-env/install_bpf_gcc.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -eu
+
+source $(cd $(dirname $0) && ${GITHUB_WORKSPACE})/../helpers.sh
```

Comment:
this was correctly using `$(pwd)`. This line essentially used to be able to find the `helpers.sh` file in the parent directory of this script. There is no warranty, and likely is is never going to be, in the parent directory of `GITHUB_WORKSPACE`.

Currently, this code is not ran because we do not `bpf-toolchain` is not set in the test cases.

Reply:
Just patched the pwd regression.

> Currently, this code is not ran because we do not `bpf-toolchain` is not set in the test cases.

Honestly, I would not know where to start to add it in.

Reply:
https://github.com/libbpf/ci/blob/f1bc702dc76510c4b306a76d1a00db5dc29fb99b/.github/workflows/test.yml#L20-L23

This is where you would want to modify the current entries to have the bpf-toolchain set to whatever the default is, and add an entry where bpf-toolchain is set to GCC.

You will then need to pass the toolchain through the with statement: https://github.com/libbpf/ci/blob/f1bc702dc76510c4b306a76d1a00db5dc29fb99b/.github/workflows/test.yml#L33

and probably need to propagate it in the reusable workflows that are indirectly used.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#using-a-matrix-strategy may help in understanding how this works.

Reply:
IIRC, you also need to run a separate set of tests for gcc, so you may need more tweaking.